README.md

```diff
@@ -212,6 +212,23 @@ sending the message to syslog.
 
 [auditd.conf(5)](https://man7.org/linux/man-pages/man5/auditd.conf.5.html)
 
+### ./defaults/main/automatic_updates.yml
+
+```yaml
+automatic_updates:
+  enabled: true
+  reboot: false
+```
+
+`automatic_updates.enabled: true` will install and configure
```

Comment:
One interesting thing to note (that I did not remember)
```
    - name: Ensure role directory for konstruktoid.hardening does not exist
      ansible.builtin.file:
        path: "{{ lookup('env', 'HOME') }}/.ansible/roles/konstruktoid.hardening"
        state: absent
      delegate_to: localhost
      run_once: true

    - name: Create the empty directory
      ansible.builtin.file:
        path: "{{ lookup('env', 'HOME') }}/.ansible/roles/konstruktoid.hardening"
        state: directory
        mode: '0755'
      delegate_to: localhost
      run_once: true

    - name: Clone hardening repository
      ansible.builtin.git:
        repo: https://github.com/konstruktoid/ansible-role-hardening.git
        dest: "{{ lookup('env', 'HOME') }}/.ansible/roles/konstruktoid.hardening"
        version: 'master'
      delegate_to: localhost
      run_once: true

    - name: Include the hardening role
      ansible.builtin.include_role:
        name: konstruktoid.hardening
      vars:
        # noqa: var-naming[no-role-prefix]
        automatic_updates.enabled: true #---> invalid variable name
        automatic_updates.reboot: true
       ....
```
only this will in my setup: 
```
        automatic_updates:
          enabled: true
          reboot: true
```

Reply:
Ah, yes. That should probably be rewritten. I meant to write the var as simple as possible, but I see now how it could be confusing.

```yaml
---
- name: Various Ansible tests
  hosts: localhost
  any_errors_fatal: true
  gather_facts: false
  vars:
    automatic_updates:
      enabled: true
      reboot: true
  tasks:
    - name: Print automatic updates
      ansible.builtin.debug:
        var: automatic_updates

    - name: Get automatic updates status
      ansible.builtin.debug:
        msg: "enabled: {{ automatic_updates.enabled }}, reboot: {{ automatic_updates.reboot }}"
```

Reply:
https://github.com/konstruktoid/ansible-role-hardening/pull/714